mev-rs/src/relay.rs

```diff
@@ -29,6 +29,21 @@ impl TryFrom<Url> for RelayEndpoint {
     }
 }
 
+impl hash::Hash for RelayEndpoint {
+  fn hash<H: hash::Hasher>(&self, state: &mut H) {
```

Comment:
nitpick but I prefer to import everything to a single symbol

so this would be 

`use std::hash::{Hash, Hasher};` at the top of this file